seed/seedingScript.js

```diff
@@ -1,14 +1,14 @@
 const fs = require('fs');
 const path = require('path');
 const { generateCSV, generateJSON } = require('./generateData');
-const { createHeaderImageRecord, createMainImageRecords } = require('./Page.js');
+const { createHeaderImageRecord, createMainImageRecords, createCouchRecord } = require('./Page.js');
 const filePath = path.join(__dirname, '/data');
 
 try {
   const headerImagesStream = fs.createWriteStream(filePath + '/headerImages.csv');
   generateCSV(headerImagesStream, createHeaderImageRecord, 1e7, () => {
-    headerImagesStream.end()
-    console.log('Successfully wrote headerImage records to CSV')
+    headerImagesStream.end();
+    console.log('Successfully wrote headerImage records to CSV');
   });
 } catch (err) {
```

Comment:
catch (err) 
  if (err) ... 

I think you can just log that err, if its caught it's an error i'm pretty sure, i'm sure this was just an oversight though